esmvalcore/cmor/_fixes/native6/era5.py

```diff
@@ -111,6 +110,7 @@ def fix_metadata(self, cubes):
             fix_hourly_time_coordinate(cube)
             fix_accumulated_units(cube)
             multiply_with_density(cube)
+            cube.data = cube.core_data() * -1.0
```

Comment:
Could you maybe add a one-line comment below "Set input cube units for invalid units were ignored on load" for the change of sign? Maybe:
`# Correct sign to align with CMOR standards`
(and maybe same for the second variable fixed)